api/environments/identities/views.py

```diff
@@ -132,6 +133,14 @@ def get_queryset(self):
 
         return queryset
 
+    def get_permissions(self):
```

Comment:
This line(https://github.com/Flagsmith/flagsmith/pull/1029/files#diff-55e28274d29ce084198b7aacadcd41d63b764f7adea2d1be9d6b15f19da96907L111) is only used for  documentation now, right?

Reply:
That link doesn't seem to take me to the correct line but I imagine you were trying to link to the `permission_classes` attribute on the view? I think I've removed that now.